tsconfig.json

```diff
@@ -0,0 +1,82 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    "incremental": true,                            /* Enable incremental compilation */
+    "target": "ES2019",                             /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "module": "ES2020",                             /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    "lib": [
+      "ES2020",
+      "ScriptHost",
+      "WebWorker",
+      "DOM",
+      "DOM.Iterable"
+    ],                                            /* Specify library files to be included in the compilation. */
+    // "allowJs": true,                             /* Allow javascript files to be compiled. */
+    // "checkJs": true,                             /* Report errors in .js files. */
+    // "jsx": "preserve",                           /* Specify JSX code generation: 'preserve', 'react-native', 'react', 'react-jsx' or 'react-jsxdev'. */
+    "declaration": true,                            /* Generates corresponding '.d.ts' file. */
+    "declarationMap": true,                         /* Generates a sourcemap for each corresponding '.d.ts' file. */
+    "sourceMap": true,                              /* Generates corresponding '.map' file. */
+    // "outFile": "./",                             /* Concatenate and emit output to single file. */
+    "outDir": "./dist",                              /* Redirect output structure to the directory. */
+    // "rootDir": "./src",                          /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    "composite": true,                           /* Enable project compilation */
+    "tsBuildInfoFile": ".tsbuildinfo",                     /* Specify file to store incremental compilation information */
+    "removeComments": true,                         /* Do not emit comments to output. */
+    // "noEmit": true,                              /* Do not emit outputs. */
+    "importHelpers": true,                          /* Import emit helpers from 'tslib'. */
+    // "downlevelIteration": true,                  /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
+    // "isolatedModules": true,                     /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
+
+    /* Strict Type-Checking Options */
+    "strict": true,                                 /* Enable all strict type-checking options. */
+    "noImplicitAny": true,                       /* Raise error on expressions and declarations with an implied 'any' type. */
```

Comment:
why `"noImplicitAny": true,` uncomment ??

Reply:
We want to use typescript rules in a strict framework

Reply:
So our variables should not be in the state of anything

Reply:
A few codes like below, maybe not work in this config. it's better handled by Eslint that we fix it in another pull request.

```
(x as any).inssert()
```
comment this line.